hack/build-container.sh

```diff
@@ -4,4 +4,5 @@ set -e
 
 source hack/common.sh
 
+cp $PROMETHEUS_RULE_PATH/*rules*.yaml $OUTDIR
```

Comment:
```suggestion
cp $PROMETHEUS_RULES_PATH/*rules*.yaml $OUTDIR
```
NIT, Since everywhere we call them rules.  Also, change in other places

Reply:
+1